docs/static/CNAME

```diff
@@ -0,0 +1 @@
+docs.urturn.app
```

Comment:
The CNAME file is important and required by gh-pages in order to properly know what the custom domain is (in our case it is `docs.urturn.app`).

Source:
https://docusaurus.io/docs/deployment#deploying-to-github-pages
>In case you want to use your custom domain for GitHub Pages, create a CNAME file in the static directory. Anything within the static directory will be copied to the root of the build directory for deployment. When using a custom domain, you should be able to move back from baseUrl: '/projectName/' to baseUrl: '/', and also set your url to your custom domain.